lib/matplotlib/tests/test_triplot_with_ls.py

```diff
@@ -0,0 +1,8 @@
+from matplotlib.testing.decorators import check_figures_equal
```

Comment:
We don't typically have one-issue test files.  Please find where triplot is tested and move there.  Thanks!

Reply:
Moved the test to _lib/matplotlib/tests/test_triangulation.py_ test file and improved formatting.
Please Review.
Thanks.